MembraneVideoroomDemo/Controllers/RoomController.swift

```diff
@@ -393,15 +457,54 @@ extension RoomController: MembraneRTCDelegate {
         }
     }
 
-    func onTrackUpdated(ctx _: TrackContext) {}
+    func onTrackUpdated(ctx: TrackContext) {
+        let isActive =
+            (ctx.metadata["active"] != nil)
+            ? ctx.metadata["active"] as! Bool : ctx.metadata["type"] as! String == "camera" ? false : true
```

Comment:
a bit complicated, can't we split it somehow?

Reply:
Extracted one new helper function, idk how make it more simple